### PR TITLE
feat: Add integration testing

### PR DESCRIPTION
**Pull request by elliott-with-the-longest-name-on-github**
Adds integration tests for all packages in this app.

TODO:
- [x] README update for Next app and root
- [x] Add environment variables necessary for tests (we need to be able to connect to the pg database)
- [ ] Fast-follow -- add Curated Tests project to this repository (can't do it without `packages/integration` on main)

### CODE REVIEW

File: packages/integration/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Instructions:
+
+- `cd packages/integration`
+- `vc link` => link to `Vercel Labs` => `vercel-storage-integration-test-suite`
```
Comment (dferber90) [inline packages/integration/README.md]: we have a curated-tests team that we should probably use here

File: packages/integration/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+module.exports = {
```
Comment (dferber90) [inline packages/integration/.eslintrc.js]: should we organize the folders differently?

I'd love if we had `packages` as the actual storage packages and other folders for supporting packages, examples and integration tests

That way people checking out this repo on github won't get overwhelmed, and it's also easier for us to see which of these are actually published to npm

Comment (elliott-with-the-longest-name-on-github) [reply packages/integration/.eslintrc.js]: How about:
- `packages` - maps `@vercel/{name}` to `packages/{name}`
- `tooling` - eslint + tsconfig + others as necessary
- `test` - integration tests

Probably good to put these in `test/next` anyway -- at some point we should add similar tests for a SvelteKit (or other Vite-based framework) app -- those would've caught our issue with the `Proxy`-as-default-export before we went live.

Comment (dferber90) [reply packages/integration/.eslintrc.js]: Sounds great

Comment (elliott-with-the-longest-name-on-github) [reply packages/integration/.eslintrc.js]: done

**Review by dferber90 — APPROVED** [review]

**Pull request merged**